approval_frame/urls.py

```diff
@@ -11,5 +10,7 @@
     url(r'^approval_polls/', include('approval_polls.urls', namespace="approval_polls")),
     url(r'^admin/', include(admin.site.urls)),
     url(r'^accounts/register/$', RegistrationView.as_view(form_class=RegistrationFormCustom), name='registration_register'),
+    url(r'^accounts/', include('registration.auth_urls')),
     url(r'^accounts/', include('registration.backends.default.urls'))
```

Comment:
Ok, cool. I didn't know if backend.default.urls had all the routes covered by registration.auth_urls, so I just wanted to confirm. Thanks!